components/home/homepage-menu.tsx

```diff
@@ -19,11 +19,22 @@ export function HomePageMenu() {
               className: "h-6 w-6 text-brand-500",
             })}
           </div>
-          <Link href={item.href}>
-            <a className="text-gray-900 font-semibold text-xs mt-3 helper-link-cover">
+          {item.external ? (
+            <a
+              className="text-gray-900 font-semibold text-xs mt-3 helper-link-cover cursor-pointer"
+              href={item.href as string}
+              rel="noopener noreferrer nofollow"
+              target="_blank"
+            >
               {item.name}
             </a>
-          </Link>
+          ) : (
+            <Link href={item.href}>
+              <a className="text-gray-900 font-semibold text-xs mt-3 helper-link-cover">
```

Comment:
Why we are not add `cursor-pointer` also in here. It should be identical

Reply:
somehow, it already has a cursor-pointer. 
I assume it's from the next link tag.

Reply:
![Screen Shot 2021-08-12 at 20 56 09](https://user-images.githubusercontent.com/7221389/129209697-8b0fff73-6174-498d-88a6-775bdc36a9f7.png)

Reply:
but also confused. 
why it has a cursor-pointer. 
![image](https://user-images.githubusercontent.com/1397612/129209991-785821fd-7bcd-4948-9ef3-e442962d60fc.png)
I check here, but there's no CSS pointer 🧐

Reply:
`helper-link-cover` is used to cover the entire list item with the link. That's the intended behaviour, since wrapping only the link text with `<a>` is better in terms of a11y.